apps/dapp-console/app/console/components/ProjectIconLinks.tsx

```diff
@@ -3,39 +3,62 @@
 import { externalRoutes } from '@/app/constants'
 import { motion } from 'framer-motion'
 import Image from 'next/image'
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@eth-optimism/ui-components/src/components/ui/tooltip'
+
+import {
+  Dialog,
+  DialogContent,
+  DialogFooter,
+  DialogTrigger,
+} from '@eth-optimism/ui-components/src/components/ui/dialog'
+import { Button } from '@eth-optimism/ui-components/src/components/ui/button'
+import { Text } from '@eth-optimism/ui-components/src/components/ui/text'
 
 const linkItems = [
```

Comment:
Would we want to put this in a constants file?

Reply:
moved it there in a later PR